### ISSUE TITLE
[Enhancement] Developer request + docs update

### ISSUE THREAD

**Issue by kerkenit** [athombv/homey#1282]
**What is your Homey version (Settings → System)?**

1.1.2

**What did you try to achieve?**

Develop an app with a flow card control witch this JSON
```
"args": [{
				"name": "itemcount"
				"type": "range",
				"min": 1,
				"max": 20,
				"step": 1,
				"label": {
					"en": "News items",
					"nl": "Nieuws berichten"
				},
				"labelDecimals": 0
			}]
```

**What was the result?**

The output of the label in the flow is 10[object Object] were 10 is the valid number.

The result is in the flow 
**What did you expect as result?**

I'm expecting if I can give a language object it shows me the label of the current language.

**Any other remarks**

Conform the docs the label is a number and the value is % (The western numeric table has no % only 1 2 3 4 5 6 7 8 9 0)
I've tried it also this and the works. **So please update the [docs](https://developers.athom.com/library/flow/arguments/)**
```
"args": [{
				"name": "itemcount"
				"type": "range",
				"min": 1,
				"max": 20,
				"step": 1,
				"label": " Nieuws berichten",
				"labelDecimals": 0
			}]
```

**Comment by caseda** [comment]
I would just read the documentation beter, before trying to blame documentation. 

It is title not label
It is number, not range
It is decimal, not labelDecimals

**Comment by WeeJeWel** [comment]
The docs were correct, it's just a bug in Homey. Fixed in 1.2.0, the correct format is `label.en`
